dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'the_tuva_project'
-version: '0.5.4'
+version: '0.5.5'
```

Comment:
@sarah-tuva We haven't released 0.5.4 yet so we can add this PR.  Can you change the version number back?

Reply:
@thutuva Good catch. Just rolled that back. CI testing is running now.